sky/packages/sky/lib/widgets/scrollable.dart

```diff
@@ -188,12 +195,12 @@ abstract class Scrollable extends StatefulComponent {
   }
 
   EventDisposition _handleScrollUpdate(sky.GestureEvent event) {
-    scrollBy(direction == ScrollDirection.horizontal ? event.dx : -event.dy);
+    scrollBy(scrollDirection == ViewportScrollDirection.horizontal ? event.dx : -event.dy);
```

Comment:
What if scrollDirection is both?  

Maybe scrollBy() and scrollTo() should have an Offset valued parameter.